apps/admin/backend/src/app.ts

```diff
@@ -931,7 +932,9 @@ function buildApi({
             isTestMode,
             writeInCandidates,
             machineConfig: getMachineConfig(),
-          })
+          }),
+          null,
+          2
```

Comment:
Is this just for debugging? Or did you want the exported JSON to be more readable? It could inflate the size a good amount

Reply:
I wanted the exported JSON to be more readable, but I don't feel strongly about it.